spec.html

```diff
@@ -65,7 +119,7 @@ <h1>Agents</h1>
             <ins>[[AsyncContextMapping]]</ins>
           </td>
           <td>
-            <ins>a List of Records with fields [[AsyncContextKey]] (a Symbol) and [[AsyncContextValue]] (an ECMAScript language value)</ins>
+            <ins>a List of Async Context Mapping Records</ins>
           </td>
           <td>
             <ins>A map from the AsyncContext's key symbol to the saved ECMAScript language value. Every Record in the List contains a unique [[AsyncContextKey]].</ins>
```

Comment:
```suggestion
            <ins>A map from AsyncLocal instances to the saved ECMAScript language value. Every Record in the List contains a unique [[AsyncContextKey]].</ins>
```

Reply:
Also, this should probably mention that the map is initially empty. Otherwise it could be taken to be filled at the host/implementation discretion, like most other agent record fields.